pages/common/sleep.md

```diff
@@ -7,10 +7,14 @@
 
 `sleep {{seconds}}`
 
-- Delay in minutes:
+- Delay in [m]inutes. (Other units [d]ay, [h]our, [s]econd, [inf]inity can also be used):
 
 `sleep {{minutes}}m`
 
-- Delay in hours:
+- Delay for 1 [d]ay 3 [h]ours:
 
-`sleep {{hours}}h`
+`sleep 1d 3h`
+
+- Execute a specific command after 20 [m]inutes delay:
+
+`sleep 20m && {{command}}`
```

Comment:
```suggestion
```

Optional fix: remove example as it's a general POSIX-compatible shell syntax and not specific to this command.